resources/views/pages-legacy/leaderboardinfo.blade.php

```diff
@@ -2,12 +2,17 @@
 
 use App\Community\Enums\ArticleType;
 use App\Enums\Permissions;
+use App\Models\Game;
+use App\Models\Leaderboard;
 use App\Platform\Enums\ValueFormat;
 use App\Platform\Services\TriggerDecoderService;
+use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\Blade;
 
 authenticateFromCookie($user, $permissions, $userDetails);
 
+$userModel = Auth::user();
```

Comment:
What's the different between `Auth::user()` and `request()->user()`? I see both used in the code.

Reply:
Had the same question myself and went down a rabbit hole on the subject recently.

For our purposes, my perception is they're both equivalent.

Functionally, `Auth::user()` retrieves the currently authenticated user based on the default guard specified in the Laravel app. `request()->user()` retrieves the currently authenticated user associated with the request.

In the context of using these in a Blade template, they're both wrappers of the same functionality. `Auth::user()` may be the more "Laravel-native" way of doing things, but I don't have a strong preference towards either.